backend/benefit/applications/models.py

```diff
@@ -881,3 +882,34 @@ class ReviewState(models.Model):
 class AhjoSetting(TimeStampedModel):
     name = models.CharField(max_length=255, unique=True)
     data = JSONField()
+
+
+class AhjoStatus(TimeStampedModel):
+    """
+    Ahjo status of the application
+    """
+
+    status = models.CharField(
+        max_length=64,
+        verbose_name=_("status"),
+        choices=AhjoStatus.choices,
+        default=AhjoStatus.SUBMITTED_BUT_NOT_SENT_TO_AHJO,
+        db_index=True,
+    )
+    application = models.ForeignKey(
+        Application,
+        verbose_name=_("application"),
+        related_name="ahjo_status",
+        on_delete=models.CASCADE,
+    )
+
+    def __str__(self):
+        return self.status
+
+    class Meta:
+        db_table = "bf_applications_ahjo_status"
+        verbose_name = _("ahjo status")
+        verbose_name_plural = _("ahjo statuses")
+        ordering = ["application__created_at", "created_at"]
+        get_latest_by = "created_at"
+        unique_together = ["application_id", "status"]
```

Comment:
> Use [UniqueConstraint](https://docs.djangoproject.com/en/4.2/ref/models/constraints/#django.db.models.UniqueConstraint) with the [constraints](https://docs.djangoproject.com/en/4.2/ref/models/options/#django.db.models.Options.constraints) option instead.
> 
> [UniqueConstraint](https://docs.djangoproject.com/en/4.2/ref/models/constraints/#django.db.models.UniqueConstraint) provides more functionality than unique_together. unique_together may be deprecated in the future.

<https://docs.djangoproject.com/en/4.2/ref/models/options/#django.db.models.Options.unique_together>

Reply:
Thx did not now about this.